cluster/aws/util.sh

```diff
@@ -190,6 +190,16 @@ function detect-master() {
   echo "Using master: $KUBE_MASTER (external IP: $KUBE_MASTER_IP)"
 }
 
+# Reads kube-env metadata from master
+#
+# Assumed vars:
+#   KUBE_MASTER_IP
+#   AWS_SSH_KEY
+#   SSH_USER
+function get-master-env() {
+  ssh -oStrictHostKeyChecking=no -i "${AWS_SSH_KEY}" ${SSH_USER}@${KUBE_MASTER_IP} sudo cat /etc/kubernetes/kube_env.yaml
```

Comment:
Yes - it's because all AWS images have a non-root account which you SSH in to, but that user is set up to allow passwordless sudo.  (Because there's no way to communicate a password).  At least all the images I've ever seen!